### ISSUE TITLE
[solved] MuMuDVB Crash with error 10 and malloc fail in a specific transponder only

### ISSUE THREAD

**Issue by mthqwork** [braice/MuMuDVB#119]
Hello there,

First a little introduce, I'm from Hungary, sorry for my poor english. 
I have an issue with a specific transponder with DVB-C tuners. 
I have 3 of TechniSat CableStar HD2 cards (mantis core driver) and, one TBS6221 T2/T/C card.

Issue is only with one TP, 778 MHz of hungarian cable provider called DIGI Cable. 
Another TP's are fine, as far as i know, only this one makes mumu to crash.

I will attach a log of "mumudvb -d -v -c conf" :

I have tried to turn off sdt, pat, eit rewriting, but crashes no matter if it's on or off. I also tried to config only one of channels with autoconf pids, still crash.
I have an idea that a strange character in one channel name makes the crash, so i include a channel scan from scan util.

[log.txt](https://github.com/braice/MuMuDVB/files/244293/log.txt)
[config.txt](https://github.com/braice/MuMuDVB/files/244294/config.txt)
[scan.txt](https://github.com/braice/MuMuDVB/files/244295/scan.txt)

**Comment by braice** [comment]
Hi

Strange, it says it cannot allocate enough memory (see errors.h), can you
run in ver very very verbose ? ( "-vvv" on the command line) and give me
the logs
I need to see who is eating the memory

Thank you

2016-05-01 15:49 GMT-04:00 mthqwork notifications@github.com:

**Comment by mthqwork** [comment]
Hello,

Thanks. I've attached the log.

[log-vvv.txt](https://github.com/braice/MuMuDVB/files/255637/log-vvv.txt)

**Comment by braice** [comment]
Hello

I have pushed a version with more debug info, I still don't understand

Can you send me again logs with the new version, and you can set all
rewrite to 0

Thank you

2016-05-09 14:31 GMT-04:00 mthqwork notifications@github.com:

**Comment by mthqwork** [comment]
Sorry to tell that, but i have compiled this version and, this version does NOT crash.
I have double checked, installed the old version again, crash, new version no crash. 
Anyway, I have attached the log. I've changed the config a bit but no sense - old crash, new works.

After a minute i hit ctrl+c, You will see at the log, then mumu just closed normally.

[new-conf.txt](https://github.com/braice/MuMuDVB/files/264904/new-conf.txt)
[new-log.txt](https://github.com/braice/MuMuDVB/files/264905/new-log.txt)
